recipes/yaclib/all/patches/0001-Set-cpp17-as-default-standard.patch

```diff
@@ -0,0 +1,25 @@
+From 34f9a2dd6232df38213121a135f2ce2ef1d9d598 Mon Sep 17 00:00:00 2001
+From: Ilya Kazakov <i.kazakov@yadro.com>
+Date: Thu, 3 Nov 2022 22:27:50 +0300
+Subject: [PATCH] Set default standard to cpp17
+
+---
+ CMakeLists.txt | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 13c28b2..b30f488 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -14,6 +14,8 @@ endif ()
+ 
+ if (YACLIB_CXX_STANDARD)
+   set(CMAKE_CXX_STANDARD ${YACLIB_CXX_STANDARD})
++else ()
++  set(CMAKE_CXX_STANDARD 17)
+ endif ()
```

Comment:
Also better to make PR to yaclib

```suggestion
+elif (NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
 endif ()
```
or maybe, not sure how conan works
```suggestion
+elif (CMAKE_CXX_STANDARD < 17)
+  set(CMAKE_CXX_STANDARD 17)
 endif ()
```

Also maybe 20 better as default, because of YACLIB_FLAGS="CORO" for many compilers requires 20 standard

```suggestion
+elif (NOT CMAKE_CXX_STANDARD AND NOT CORO IN_LIST YACLIB_FLAGS)
+  set(CMAKE_CXX_STANDARD 17)
+elif (NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 20)
 endif ()
```

Reply:
I'll make PR to YAClib in the nearest future